src/features/Settings/components/AddOrEditNetworkNode.tsx

```diff
@@ -254,19 +296,58 @@ export default function AddOrEditNetworkNode({
           ) as CustomNodeConfig;
 
           try {
-            const network = getNetworkById(selectedNetworkId);
+            const network: Network = !isAddingCustomNetwork
+              ? getNetworkById(selectedNetworkId)
+              : {
+                  id: selectedNetworkId,
+                  name: networkName!,
+                  chainId: parseInt(chainId!, 10),
+                  baseUnit: baseUnit as TTicker,
+                  baseAsset: generateAssetUUID(chainId!),
+                  isCustom: true,
+                  assets: [],
+                  contracts: [],
+                  nodes: [node],
+                  dPaths: {
+                    [WalletId.TREZOR]: DPaths.ETH_TESTNET,
+                    [WalletId.LEDGER_NANO_S]: DPaths.ETH_TESTNET,
+                    [WalletId.MNEMONIC_PHRASE]: DPaths.ETH_TESTNET
```

Comment:
Are you sure those DPaths are the one to use ?

Reply:
Hmm, do we just want to use the default ones?

Reply:
Let me check the one used with ganache

Reply:
Looks to be `m/44'/60'/0'/0/{account_index}` for Ganache. Which is `ETH_DEFAULT`

Reply:
Yup ! You're right :) I've got the same